x-pack/test/fleet_api_integration/apis/epm/install_by_upload.ts

```diff
@@ -10,12 +10,15 @@ import path from 'path';
 import expect from '@kbn/expect';
 import { INGEST_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
 import { HTTPError } from 'superagent';
+import { promisify } from 'util';
 
 import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
 import { skipIfNoDockerRegistry, isDockerRegistryEnabledOrSkipped } from '../../helpers';
 import { setupFleetAndAgents } from '../agents/services';
 import { testUsers } from '../test_users';
 
+const sleep = promisify(setTimeout);
```

Comment:
nit: there is a builtin API in node that does that `import { setTimeout } from 'node:timers/promises';`